nion/eels_analysis/BackgroundModel.py

```diff
@@ -120,7 +120,43 @@ def __unused_perform_fit(self, xs: numpy.ndarray, ys: numpy.ndarray, fs: numpy.n
         return untransform_data(series(fs))
 
 
+class TwoAreaBackgroundModel(AbstractBackgroundModel):
+#Fit power law or exponential background model using the two-area method described in Egerton chapter 4.
+#This approximation is slightly faster than the polynomial fit for mapping large SI, and may perform better for high-noise spectra.
```

Comment:
Sorry - one more minor formatting fix: comments in Python follow a specific convention (aka PEP 8). Can you format this comment as a block comment, indented at the level of 'def' below it?

See https://www.python.org/dev/peps/pep-0008/#id33

Reply:
Btw - it's worth reading over the entire PEP 8 at some point.

Reply:
Thanks, will do. I think I've looked at that before, but it was some time ago.